index.html

```diff
@@ -227,9 +225,9 @@ <h2>
         </li>
       </ol>
       <p>
-        When the <a>user agent</a> determines that the <a href=
-        "https://w3c.github.io/page-visibility/#dfn-visibility-states">visibility
-        state</a> of the <code>Document</code> of the <a>top-level browsing
+        When the <a>user agent</a> determines that 
+        the {{Document/visibilityState}} of 
```

Comment:
There's the same issue here.

```suggestion
        the [=Document/visibility state=] of 
```